packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -101,27 +110,48 @@ async function main() {
   const proposerV2 = await proposerFactory.deploy(votingToken.address, defaultBond, governorV2.address, finder.address);
   console.log("Deployed ProposerV2: ", proposerV2.address);
 
-  console.log("6. Set ProposerV2 as the proposer of GovernorV2");
+  console.log("6. Deploying EmergencyProposer");
+  const quorum = hre.web3.utils.toWei("10000000", "ether");
```

Comment:
Should this be a config we pass in?

Reply:
This is a cool idea. Expanding on this a bit, we could have a json file represent a full simulation scenario that could mix and match different scripts with different parameters to each. Similar to what we have in the bots. Might be overkill, but it could be a cool way to do multi-script simulations going forward.